garrysmod/lua/vgui/dframe.lua

```diff
@@ -16,12 +16,15 @@ local PANEL = {}
 AccessorFunc( PANEL, "m_bIsMenuComponent", 		"IsMenu", 			FORCE_BOOL )
 AccessorFunc( PANEL, "m_bDraggable", 			"Draggable", 		FORCE_BOOL )
 AccessorFunc( PANEL, "m_bSizable", 				"Sizable", 			FORCE_BOOL )
+AccessorFunc( PANEL, "m_bMinMaxAnim", 			"MinMaxAnim", 		FORCE_BOOL )

```

Comment:
You mean indentation style?  But yes, you should change them to use spaces for alignment rather than tabs.
The reason this isn't done altogether is because it can break merges, but if you're going to edit the file and its alignments, then you should fix it and kill two birds with one stone.